src/components/interaction-designer/flow-editors/ModesEditor.vue

```diff
@@ -15,10 +15,12 @@
 import VueMultiselect from 'vue-multiselect'
 import {Component, Prop} from 'vue-property-decorator'
 import {IFlow, SupportedMode} from '@floip/flow-runner'
-
 import Lang from '@/lib/filters/lang'
 import {mixins} from 'vue-class-component'
-import {Getter} from 'vuex-class'
+import {namespace, State} from 'vuex-class'
+import {orderModes} from '@/store/flow/flow'
```

Comment:
`orderModes` - modes are stored in unsorted form, but we sort them here - on the surface, in the UI - to display in certain order